packages/commerce-sdk-react/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "commerce-sdk-react-preview",
-  "version": "2.7.4",
+  "version": "2.8.0-preview.0",
```

Comment:
Do we want to release a new version of `commerce-sdk-react-preview`?

Knowing that the final version is living on NPM `@salesforce/commerce-sdk-react`

Reply:
@adamraya this will not be published because the package is marked as `private: true`.